earth_enterprise/rpms/buildSrc/src/main/groovy/GeeDeb.groovy

```diff
@@ -134,6 +136,10 @@ class GeeDeb extends com.netflix.gradle.plugins.deb.Deb {
         packageDescription = formattedDescription
     }
 
+    // Variable to autodetect symlinks. Detected symlinks are fixed, so they
+    // are not created as files.
+    boolean autoSymlinkDetection = true
```

Comment:
Minor: I think `autoSymlinkDetection` is not very informative of the effect you get when you set it to `true`.  I think a better name may be something like `preserveSymlinks`.

Reply:
Changed.